plugins/idamagic/helpers.py

```diff
@@ -798,7 +811,8 @@ def parse_sib(op_t):
 
 def get_ida_version():
     try:
-        version_string = idaapi.IDA_SDK_VERSION
+        # String response is formated "x.y(.z)", e.g. "8.4(.2)"
+        version_string = ida_kernwin.get_kernel_version()
```

Comment:
This doesn't seem to tell us the patch version, but I think that's probably ok.

Also, the below dotted_version is no longer needed with this, we can probably just 
`return ida_kernwin.get_kernel_version()` here.

Check within the ida python terminal

Reply:
That makes sense. I don't know how it would impact the disassembly upload, since we pass this dict with the upload:
```
        bin_dict = {
            "md5": get_ida_md5(),
            "sha1": disassembly_hashes["sha1"],
            "sha256": get_ida_sha256(),
            "sha512": disassembly_hashes["sha512"],
            "unix_filetype": getUnixFileType(),
            "version": get_ida_version(),
            "disassembler": "ida",
	        "use_32": arch == "32-bit",
	        "use_64": arch == "64-bit",
	        "file_name": get_input_file_name(),
            "image_base": get_image_base(),
            "byte_data": convert_to_encoded_byte_string(),
        }
```

If not having the patch version breaks this, I can revert the change here.